scripts/test-turbo-modules.sh

```diff
@@ -165,7 +165,7 @@ create_library() {
     example_type=vanilla
   fi
   echo "-- Creating library $PROJECT_SLUG with create-react-native-library@$BOB_VERSION"
-  npx "create-react-native-library@$BOB_VERSION" \
+  npm_config_yes=true npx "create-react-native-library@$BOB_VERSION" \
```

Comment:
This confirms the install prompt when you haven't run the supplied version of create-react-native-library before.

```
Need to install the following packages:
create-react-native-library@0.43.0
Ok to proceed? (y)
```